upup/pkg/fi/cloudup/template_functions.go

```diff
@@ -659,23 +659,30 @@ func (tf *TemplateFunctions) OpenStackCCMTag() string {
 	return tag
 }
 
-// AWSCCMTag returns the correct tag for the cloud controller manager based on
+// AWSCCMImage returns the correct image for the cloud controller manager based on
 // the Kubernetes Version
-func (tf *TemplateFunctions) AWSCCMTag() (string, error) {
+func (tf *TemplateFunctions) AWSCCMImage() (string, error) {
```

Comment:
Maybe we should move this to pkg/model/components/awscloudcontrollermanager.go with the other arguments?
then we only need `{{ .ExternalCloudControllerManager.Image }}` in the template.

Reply:
I think @rifelpet did that in https://github.com/kubernetes/kops/pull/12104?